strider/compatibility.py

```diff
@@ -180,6 +180,10 @@ def add_source(message: Message, kp_id):
                 "upstream_resource_ids": [kp_id],
             }
         )
+    for result in message.get("results", []):
+        for analysis in result.get("analyses", []):
+            analysis["resource_id"] = "infores:aragorn"
+            analysis["score"] = None
```

Comment:
I would put a note here maybe saying that we end up clearing out any scores given to us by KPs.